openedx/core/djangoapps/team_api/tests/test_views.py

```diff
@@ -0,0 +1,437 @@
+"""Tests for the teams API at the HTTP request level."""
+# pylint: disable=maybe-no-member
+import json
+
+from django.conf import settings
+from django.core.urlresolvers import reverse
+from rest_framework.test import APITestCase
+from unittest import skipUnless
+
+from student.tests.factories import UserFactory
+from student.models import CourseEnrollment
+from xmodule.modulestore.tests.factories import CourseFactory
+from .factories import CourseTeamFactory
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+
+
+@skipUnless(settings.ROOT_URLCONF == 'lms.urls', 'Team APIs are only supported in LMS')
+class TestTeamAPI(APITestCase, ModuleStoreTestCase):
+    """
+    Tests of the Team API
```

Comment:
I'm not sure. We're using the same pagination classes that are used in other places in the code base, but I guess it wouldn't hurt to add a few simple tests.